hack/versions/pkg/schema/version.go

```diff
@@ -24,16 +24,16 @@ import (
 	"strings"
 
 	"github.com/sirupsen/logrus"
+	"k8s.io/client-go/tools/clientcmd/api/latest"
 
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/update"
```

Comment:
I think this import change is unintended, it will cause issues with our version command

Reply:
fixed it!

Reply:
👍 good catch!